src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,9 +62,12 @@ public RobotContainer() {
   private void configureButtonBindings() {
     OI.Driver.getOrientationButton().onTrue(new InstantCommand(swerveSubsystem::toggleOrientation));
     OI.Driver.getZeroButton().onTrue(new InstantCommand(swerveSubsystem::zeroHeading));
+    OI.Driver.getAlignForwardPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(0), swerveSubsystem));
+    OI.Driver.getAlignBackPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(180), swerveSubsystem));
+    OI.Driver.getAlignLeftPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(90), swerveSubsystem));
+    OI.Driver.getAlignRightPOV().onTrue(new InstantCommand(() -> swerveSubsystem.enableRotationHold(-90), swerveSubsystem));
     OI.Driver.getIntakeButton().onTrue(new InstantCommand(intakeSubsystem::pull));
     OI.Driver.getOuttakeButton().onTrue(new InstantCommand(intakeSubsystem::push));
```

Comment:
Need to include the subsystem as a second parameter of InstantCommand (See [InstantCommand](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/InstantCommand.html#%3Cinit%3E(java.lang.Runnable,edu.wpi.first.wpilibj2.command.Subsystem...)) or parameters in the InstantCommand instantiations above this line)